ArcadiaParties.API/Properties/launchSettings.json

```diff
@@ -21,10 +21,12 @@
       "commandName": "Project",
       "launchBrowser": true,
       "launchUrl": "swagger",
-      "applicationUrl": "http://localhost:5000",
       "environmentVariables": {
         "ASPNETCORE_ENVIRONMENT": "Development"
-      }
+      },
+      "sqlDebugging": false,
```

Comment:
Why do you need this?

Reply:
Was automatically added after Microsoft.AspNetCore.Authentication.Negotiate NuGet package (Kestrel) installation

Reply:
Ok then.